docs/sql-guide/statements/statement-insert.md

```diff
@@ -106,10 +106,24 @@ idsetcolq idsetq timequantum 'YMD'
 The following statement inserts values with an associated timestamp (using either a unix time or timestamp) into `timeq`,  which has `IDSETQ` and `STRINGSETQ` data types.
 
 ```sql
-INSERT INTO timeq(_id, stringsetcolq, idsetcolq) 
+INSERT INTO timeq(_id, stringsetcolq, idsetcolq)
 VALUES (1, {'2018-08-31T00:00:00Z', ['A','B']}, {1676649734, [1]});
 ```
 
+### CREATE TABLE with `STRINGSET` data types
```

Comment:
This content came about because of the work on the new concept example.
There wasn't an example of inserting into a stringset data type so here it is